lib/mixpanel-node.js

```diff
@@ -35,8 +39,11 @@ var create_client = function(token, config) {
     const MAX_BATCH_SIZE = 50;
     const TRACK_AGE_LIMIT = 60 * 60 * 24 * 5;
     const REQUEST_LIBS = {http, https};
+    const REQUEST_AGENTS = {http: httpAgent, https: httpsAgent};
```

Comment:
Call me crazy, but I don't see this constant actually being used anywhere. The test case below is a bit flimsy in that regard (it just checks that an `Agent` has been initialized at some point, which you do on module import, and that requests get sent with a non-null `agent`, but the agent set on L116 is not one of these keepAlive ones).

Reply:
🙈 typo